Tools/cases_generator/generate_cases.py

```diff
@@ -283,7 +283,8 @@ def write_metadata(self, metadata_filename: str, pymetadata_filename: str) -> No
             self.out.emit(
                 "#define IS_VALID_OPCODE(OP) \\\n"
                 "    (((OP) >= 0) && ((OP) < OPCODE_METADATA_SIZE) && \\\n"
-                "     (_PyOpcode_opcode_metadata[(OP)].valid_entry))"
+                "     (_PyOpcode_opcode_metadata[(OP)].valid_entry)) || \\\n"
+                "     ((OP) == INSTRUMENTED_LINE)"   # implemented in ceval.c, not bytecodes.c
```

Comment:
I had to special-case INSTRUMENTED_LINE because it's implemented in ceval.c rather than bytecodes.c, so doesn't apprear in the metadata. We could instead do something else like add it to the instr list after bytecode.c is parsed. I'm not sure which option is less bad.